Creating a Production Image.md

```diff
@@ -1,76 +0,0 @@
-To create images for production, I set up a virtual machine in virt-manager that relies on a `raw` disk image for its disk. I usually use an Arch Linux-based host system, though Ubuntu 20 and Debian 11 also work. This outlines how to do that. After the VM is set up, the machine can be build and locked down as in the [INSTALL.md](https://github.com/votingworks/vxsuite-complete-system/blob/main/INSTALL.md) script, and the disk image flashed as described in the [vx-iso](https://github.com/votingworks/vx-iso) process. 
```

Comment:
This file and INSTALL.md below are both out of date and no longer relevant, so I'm removing them. I only found my way to them because they included some references to VxMark